README.md

```diff
@@ -118,7 +121,12 @@ $ heroku buildpacks -a yourappname
 - Go to the [Google Developers Console](https://console.developers.google.com/project) and create a new project
 - Once you've created the project, go to `APIs` and add the `Contacts API` and the `Google+ API` (you won't need a Google+ account to sign in, this is just an annoying Google quirk).
 - Then go to `Credentials` and `Create a new Client ID`. You'll need the app's production URL to complete this step so if you're using the Heroku button, do that first. You can use your production URL for the `JavaScript Origins` setting, but make sure to use `http://yourdomain.com/auth/google_oauth2/callback` for in the `Redirect URIs` setting. It's a good idea to also add the same URL but with the HTTPS protocol to ensure that if you ever force SSL, Google will still accept the redirect.
-- Don't forget to go update the `GOOGLE_KEY` and `GOOGLE_SECRET` environment variables with the credentials Google gave you when you created your Client ID, otherwise the redirection process will fail.
+- Don't forget to go update the `OAUTH_PROVIDER_KEY` and `OAUTH_PROVIDER_SECRET` environment variables with the credentials Google gave you when you created your Client ID, otherwise the redirection process will fail.
+
+### Other OAuth providers
+
+- Get the key and secret from the OAuth provider of your choice by setting up a new OAuth application with appropriate *redirect URI* and *logout redirect URI*
```

Comment:
Might be good to mention `OAUTH_PROVIDER_URL` somewhere here.

Reply:
I added a mention of this. I'll have to look into omniauth of how to use it. I know it would be used for custom OAuth identity providers. Need to look into how it works with `OmniAuth::Builder`